gradle/dependencies.gradle

```diff
@@ -18,12 +18,13 @@ def versions = [
     moshix                : '0.11.2',
     okHttp4               : '4.9.0',
     okHttp                : '3.12.11', // Keep this to keep supporting older Android devices
-    okio                  : '2.10.0',
+    okio                  : '3.0.0-alpha.8',
```

Comment:
Can you remind me why okio `3.0.0` is required for JS? It's going to be a problem if we want to ship a stable `3.0` release for the JVM.

Reply:
Because since 3.0.0-alpha.6, they support the JS IR backend.

https://github.com/square/okio/blob/master/CHANGELOG.md#version-300-alpha6

Reply:
Alright, lets cross fingers that by the time Apollo Android hits stable, Okio will be too. Else we'll have to either:
* revert to the legacy JS compiler
* or find a way to use okio 2.x for JVM and 3.x for JS